content/content-strategy/manage-content-requests/plan-content/index.yml

```diff
@@ -0,0 +1,21 @@
+pagetitle: Plan content
+description: Plan to define the end-to-end content task so the broader team has clarity about user needs and business goals.
+weight: 10
```

Comment:
Check the `weight` values against other sub topics: that they're displaying in the correct order in the side nav.